### ISSUE TITLE
Incompatible with Velocity or other proxies

### ISSUE THREAD

**Issue by Shivter14** [Alexandra-Myers/Combatify#89]
Hello, I own a minecraft server network with modded servers, but I can't use modify on a backend server because even if a player has the mod installed, the backend server refuses the connection due to it thinking that the player doesn't have the mod installed.
![image](https://github.com/user-attachments/assets/158cf166-10f0-48d7-9bc1-09c109ae6e5b)
![image](https://github.com/user-attachments/assets/83c62ab6-4b71-43f9-b28e-41e090e11133)

## More info
Minecraft Version: Fabric 1.20.1
Fabric Version: 0.16.7
Combatify Version: 1.1.8.3
Velocity Proxy Version: 3.4.0-SNAPSHOT (git-08a42b37-b449)

I am using the crossstitch mod on the server aswell, which should add additional support for Fabric modded backend servers.

**Comment by Alexandra-Myers** [comment]
Right, first off, 1.20.1, so I am not sure... If possible, could you test 1.21.4?
